metadata.rb

```diff
@@ -10,15 +10,15 @@
 
 recipe 'kafka::default', 'Downloads and installs Kafka from binary releases'
 
-suggests 'java', '~> 1.22' # ~FC052
+depends 'java', '~> 1.22'
```

Comment:
Since the metadata is rarely human read (although i read them all the time) and the java cookbook is technically not a dependency i will add a note to the README then